### PR TITLE
man: fix vrf netdev example & add vrf section

### PR DESCRIPTION
**Pull request by fpletz**
The example was wrong and a section on VRF config options was missing.

### CODE REVIEW

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -1831,7 +1831,23 @@
...
 
+  <refsect1>
+    <title>[VRF] Section Options</title>
+    <para>The <literal>[VRF]</literal> section only applies for
+    netdevs of kind <literal>vrf</literal> and accepts the
+    following key:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>TableId=</varname></term>
```
Comment (yuwata) [inline man/systemd.netdev.xml:1844]: Please use `Table=` instead. `TableId=` is deprecated.

File: man/systemd.netdev.xml [hunk 2]
```diff
@@ -1992,7 +2008,7 @@ Name=vrf-test
 Kind=vrf
 
 [VRF]
-Table=42</programlisting>
+TableId=42</programlisting>
```
Comment (yuwata) [inline man/systemd.netdev.xml:2011]: Please keep this.

**Comment by keszybz** [comment]
@fpletz ping?

**Comment by yuwata** [comment]
#12689 replaces this PR. Closing.

**Pull request closed without merging**